php.yaml

```diff
@@ -12,43 +12,178 @@ package:
 environment:
   contents:
     packages:
+      - build-base
       - busybox
+      - file
+      - bison
+      - libtool
       - ca-certificates-bundle
-      - build-base
       - bzip2-dev
       - libxml2-dev
       - curl-dev
       - openssl-dev
       - readline-dev
       - sqlite-dev
       - libsodium-dev
+      - libpng-dev
+      - libjpeg-dev
+      - libavif-dev
+      - libwebp-dev
+      - libxpm-dev
+      - libx11-dev
+      - freetype-dev
+      - gmp-dev
+      - icu-dev
+      - openldap-dev
+      - oniguruma-dev
+      - libxslt-dev
+      - postgresql-15-dev
+      - zip
 
 pipeline:
   - uses: fetch
     with:
       uri: https://www.php.net/distributions/php-${{package.version}}.tar.gz
       expected-sha256: 7046f939f0e5116285341d55c06af1d50907e107ac2c70defc32ef880f88cde4
+      extract: false
+
+  - name: Prepare Build Folder
+    runs: |
+      mv php-${{package.version}}.tar.gz $HOME/
+      tar zxf $HOME/php-${{package.version}}.tar.gz
 
   - name: Configure
     runs: |
-      ./configure \
-        --prefix=${{targets.destdir}} \
+      cd $HOME/php-${{package.version}}
+      EXTENSION_DIR=/usr/lib/php/modules ./configure \
+        --prefix=/usr \
+        --libdir=/usr/lib/php \
+        --datadir=/usr/share/php \
+        --sysconfdir=/etc/php \
+        --localstatedir=/var \
+        --with-layout=GNU \
+        --with-pic \
+        --with-config-file-path=/etc/php \
+        --with-config-file-scan-dir=/etc/php/conf.d \
         --enable-cli \
-        --with-curl \
-        --with-openssl \
+        --enable-ctype=shared \
+        --enable-bcmath=shared \
+        --with-curl=shared \
+        --enable-dom=shared \
+        --enable-fileinfo=shared \
+        --with-iconv=shared \
+        --with-openssl=shared \
         --with-readline \
+        --with-sodium=shared \
+        --enable-fpm \
+        --with-pear \
+        --enable-gd=shared \
+            --with-avif \
+            --with-freetype \
+            --with-jpeg \
+            --with-webp \
+            --with-xpm \
+            --disable-gd-jis-conv \
+        --with-libxml \
+        --enable-phar=shared \
+        --enable-mbstring=shared \
+      	--with-mysqli=shared \
```

Comment:
Weird indenting here and throughout

Reply:
It's because some of these `with-` options are linked to a parent option (such as `with-pdo-X` are dependent on `--enable-pdo`) so this is an attempt to keep the related options together (as in the [Alpine build](https://git.alpinelinux.org/aports/tree/community/php82/APKBUILD#n244)).